package.json

```diff
@@ -63,37 +62,37 @@
     "fs-extra": "10.1.0",
     "http-proxy-agent": "5.0.0",
     "https-proxy-agent": "5.0.1",
-    "jsonpath-plus": "7.0.0",
+    "jsonpath-plus": "7.2.0",
     "mailparser": "3.5.0",
     "mountebank-formatters": "0.0.2",
     "nodemailer": "6.7.8",
-    "prom-client": "14.0.1",
+    "prom-client": "14.1.0",
     "proper-lockfile": "4.1.2",
     "safe-regex": "^2.1.1",
-    "safe-stable-stringify": "2.3.1",
+    "safe-stable-stringify": "2.4.0",
     "smtp-server": "3.11.0",
-    "winston": "3.8.1",
+    "winston": "3.8.2",
     "xpath": "0.0.32",
     "yargs": "17.5.1"
   },
   "devDependencies": {
     "coveralls": "3.1.1",
-    "eslint": "8.22.0",
+    "eslint": "8.24.0",
     "eslint-plugin-node": "11.1.0",
-    "firebase-tools": "11.6.0",
+    "firebase-tools": "11.10.0",
     "jsdoc": "3.6.11",
     "jsdom": "20.0.0",
     "license-checker": "25.0.1",
     "mocha": "10.0.0",
     "mocha-multi-reporters": "1.5.1",
     "nc": "1.0.3",
     "nyc": "^15.1.0",
-    "snyk": "1.986.0",
-    "w3cjs": "0.4.0"
+    "snyk": "1.1012.0",
+    "w3cjs": "0.4.0",
+    "mbTest": "file:./mbTest"
```

Comment:
mbTest defined as development dependency in mountebank to have its modules also installed when installing mountebank locally.